tests/shell/tests.py

```diff
@@ -17,3 +19,13 @@ def test_command_option(self):
             )
             self.assertEqual(len(logger), 1)
             self.assertEqual(logger[0], __version__)
+
+    @mock.patch('django.core.management.commands.shell.select')
```

Comment:
Aha I found what you should be using for both: `django.utils.captured_stdin` and `django.utils.captured_stdout` 😉 

You could also open a second PR removing the hack from `createsuperuser` and change its tests to use `captured_stdin` 👍  Presumably